lib/wor/paginate/adapters/active_record.rb

```diff
@@ -1,10 +1,14 @@
+require_relative 'helpers/total_count'
```

Comment:
It's really necessary to use this `require_relative` ?

Reply:
Is necessary to require total_count. Is `require` preferred over to `require_relative`?

Reply:
I prefer use `require`